lua/nvim-treesitter/parsers.lua

```diff
@@ -1170,6 +1170,14 @@ list.objc = {
   maintainers = { "@amaanq" },
 }
 
+list.objdump = {
+  install_info = {
+    url = "https://github.com/ColinKennedy/tree-sitter-usd",
```

Comment:
copypasta, surely?

Reply:
Yes, sorry. I should've waited until I was less tired before submitting. This PR notwithstanding, the parser is pretty solid, thankfully, haha. Thank you for your understanding